connector_search_engine/models/se_binding.py

```diff
@@ -130,6 +130,13 @@ def jobify_recompute_json(self, force_export: bool = False):
                 description=description, identity_key=identity_exact
             ).recompute_json(force_export=force_export)
 
+    def _contextualize(self, record):
+        ctx = {"index_id": record.index_id.id}
```

Comment:
@hparfr  Why do you put only the id and not the record itself? In any case, it we need this information, we need to get access to the record...

Reply:
by mimetism, usually I see context with simple values and types. 

For instance we set the `code` instead of the lang record in the same block.

Reply:
> Agree, if there is no drawback to put the record here, I can correct it.
IMO no drawback, only advantage...

> For instance we set the code instead of the lang record in the same block.

That's a specific case since it's easier for the developer to use the code.